modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductDetails.testcase

```diff
@@ -1546,6 +1546,162 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-11650. User can checkout a product in a channel that has also stock in another channel, and the product's stock in another channel cannot be affected by this channel"
```

Comment:
"As a user, when I place an order with a product that has also stock in a second channel, then the product's stock in the second channel won't be affected by the order made in the first one."